### ISSUE TITLE
Please add an uninstaller for PeepOpen

### ISSUE THREAD

**Issue by conradwt** [topfunky/PeepOpen-Issues#46]
It would be great to add a feature to remove/uninstall PeepOpen because simply removing the application from the Applications directory doesn't work.

**Comment by topfunky** [comment]
Good point. In the meantime, you can remove the PeepOpen TextMate bundle at ~/Library/Application Support/TextMate/Bundles.

**Comment by jberkel** [comment]
it's located in ~/Library/Application Support/TextMate/PlugIns, not Bundles

**Comment by paulca** [comment]
It was Bundles for me.

**Comment by dovadi** [comment]
For some reason I had to delete the bundle in TextMate self (with the bundle editor) as well, because removing the directory from ~/Library/Application Support/TextMate/Bundles was not enough.

**Comment by judy-zz** [comment]
+1, this worked for me as well. I only had to delete the plugin.

But I would also really recommend a simple uninstalling feature. This was a PITA to figure out.
